Publisher unreachable

Happened to me on my LAN:
* macOS MBP was used first
* then moved to Linux desktop

When tried to build something already built on macOS, I ended up with strange error on Linux desktop:
```
[cstamas@angeleyes mimir (main)]$ mvn clean install
[INFO] 
[INFO] 1 problem was encountered while building the effective settings (use -e to see details)
[INFO] 
[INFO] Mimir session created with DaemonNode (socketPath=/home/cstamas/.mimir/mimir-socket; daemonPID=24568; daemonVersion=0.4.0)
[INFO] Scanning for projects...
[ERROR] Some problems were encountered while processing the POMs
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project eu.maveniverse.maven.mimir:mimir:pom:0.4.1-SNAPSHOT (/home/cstamas/Worx/maveniverse/mimir/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for eu.maveniverse.maven.mimir:mimir:0.4.1-SNAPSHOT: The following artifacts could not be resolved: eu.maveniverse.maven.parent:parent:pom:26 (absent): Could not transfer artifact eu.maveniverse.maven.parent:parent:pom:26 from/to central (https://repo.maven.apache.org/maven2/): EOFException (remote repositories: central (https://repo.maven.apache.org/maven2/, default, releases)) and parent could not be found in reactor @ eu.maveniverse.maven.mimir:mimir:0.4.1-SNAPSHOT, file:///home/cstamas/Worx/maveniverse/mimir/pom.xml, line 14, column 3: One or more requests failed
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the '-e' switch
[ERROR] Re-run Maven using the '-X' switch to enable verbose output
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[cstamas@angeleyes mimir (main)]$
```

Turned out this happened:
* desktop build started Mimir, it joined cluster (so far macOS was alone in cluster)
* desktop asked for the parent 26 from macOS, it was present (cached on macOS)
* desktop tried to get content (publisher) but publisher was unreachable

Am still unsure is this macOS related (firewall is off, but macOS did sleep a while), so am recording this only for later reference.

--- a/core/src/main/java/eu/maveniverse/maven/mimir/shared/impl/publisher/PublisherRemoteEntry.java
+++ b/core/src/main/java/eu/maveniverse/maven/mimir/shared/impl/publisher/PublisherRemoteEntry.java
@@ -30,19 +30,23 @@ public class PublisherRemoteEntry extends EntrySupport implements RemoteEntry {
     @Override
     public void handleContent(IOConsumer consumer) throws IOException {
         String schema = handle.getScheme();
-        if ("http".equals(schema)) {
-            try (InputStream inputStream = handle.toURL().openConnection().getInputStream()) {
-                consumer.accept(inputStream);
+        try {
+            if ("http".equals(schema)) {
+                try (InputStream inputStream = handle.toURL().openConnection().getInputStream()) {
+                    consumer.accept(inputStream);
+                }
+            } else if ("socket".equals(schema)) {
+                try (Socket socket = new Socket(handle.getHost(), handle.getPort())) {
+                    OutputStream os = socket.getOutputStream();
+                    os.write(handle.getPath().substring(1).getBytes(StandardCharsets.UTF_8));
+                    os.flush();
+                    consumer.accept(socket.getInputStream());
+                }
+            } else {
+                throw new IOException("Unknown protocol: " + schema);
             }
-        } else if ("socket".equals(schema)) {
-            try (Socket socket = new Socket(handle.getHost(), handle.getPort())) {
-                OutputStream os = socket.getOutputStream();
-                os.write(handle.getPath().substring(1).getBytes(StandardCharsets.UTF_8));
-                os.flush();
-                consumer.accept(socket.getInputStream());
-            }
-        } else {
-            throw new IOException("Unknown protocol: " + schema);
+        } catch (IOException e) {
+            throw new IOException("Failed to get artifact content from publisher at " + handle.toASCIIString(), e);
         }
     }
 }

